treeherder/settings/base.py

```diff
@@ -341,6 +341,11 @@
 # like ftp.mozilla.org or hg.mozilla.org
 TREEHERDER_REQUESTS_TIMEOUT = 30
 
+# the max size of a posted request to treeherder client.
+# If TreeherderCollections are larger, they will be chunked
+# to this size.
+TREEHERDER_CLIENT_CHUNK_SIZE = 50
```

Comment:
I think this should be set on a case by case basis. I guess loading 50 jobs takes not the same time as loading 50 resultsets/artifacts/other, so I don't see much value in having a centralized setting for it.